Minimum OpenSSL Version Required (#7763)
istio/istio.io: content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination-sds/index.md

    Note that any pod that you can `exec` and `curl` from would do.

> *   For macOS users, verify that you use openssl version at least 1.1:

Reviewer: ```suggestion
*   For macOS users, verify that you are using `openssl` version 1.1 or later:
```

    {{< text bash >}}
    $ openssl version -a | grep OpenSSL
    OpenSSL 1.1.1g  21 Apr 2020
    {{< /text >}}

>     If the previous command outputs a version `1.1` or later of OpenSSL as shown, your `openssl` command

Reviewer: ```suggestion
    If the previous command outputs a version `1.1` or later, as shown, your `openssl` command
```
